config/environments/development.rb

```diff
@@ -21,6 +21,9 @@
   config.action_mailer.smtp_settings = { address: ENV['SMTP_HOST'] || 'localhost', port: 1025 }
   config.action_mailer.default_options = { from: Figaro.env.email_from }
 
+  # Allow SVG's only because we will always serve them in an <img> element
+  config.active_storage.content_types_to_serve_as_binary -= ['image/svg+xml']
```

Comment:
We're not really using ActiveStorage for these right? We're just reading remotely...so we don't need this change yet?

Reply:
yeah, good call.